maas-schemas/schemas/maas-backend/customers/personal-documents/update/request.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Why not reference the create api request format. Same thing right?

Reply:
The required is different, because this is update call.
And I'm not sure how to do oneOf, allOf thingy